app/src/main/java/io/github/droidkaigi/confsched2018/data/db/dao/SessionFeedbackDao.kt

```diff
@@ -0,0 +1,25 @@
+package io.github.droidkaigi.confsched2018.data.db.dao
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.Query
+import android.arch.persistence.room.Transaction
+import io.github.droidkaigi.confsched2018.data.db.entity.SessionFeedbackEntity
+import io.reactivex.Flowable
+
+@Dao abstract class SessionFeedbackDao {
+    @Query("SELECT * FROM session_feedback")
+    abstract fun getAllSessionFeedback(): Flowable<List<SessionFeedbackEntity>>
+
+    @Query("DELETE FROM session_feedback WHERE session_id = :sessionId")
+    abstract fun delete(sessionId: String)
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    abstract fun insert(sessionFeedback: SessionFeedbackEntity)
+
+    @Transaction open fun clearAndInsert(sessionFeedback: SessionFeedbackEntity) {
```

Comment:
If you don't need to create new row id, then it's enough to use `insert`.

Reply:
yes I want to update row.
If I misunderstood the intention of the question, sorry.